crates/jstzd/src/protocol/mod.rs

```diff
@@ -81,6 +87,16 @@ impl ProtocolParameterBuilder {
         self
     }
 
+    pub fn set_bootstrap_accounts(
```

Comment:
Maybe rename to `merge_bootstrap_accounts` and add a short comment describing behaviour if the account already exists

Reply:
I decide to keep `set_bootstrap_accounts` and change its behaviour to replacing the current set of accounts. It makes more sense for this operation not to keep anything from previous calls.